validator_client/src/beacon_node_fallback.rs

```diff
@@ -32,6 +38,18 @@ const UPDATE_REQUIRED_LOG_HINT: &str = "this VC or the remote BN may need updati
 /// having the correct nodes up and running prior to the start of the slot.
 const SLOT_LOOKAHEAD: Duration = Duration::from_secs(2);
 
+/// If the beacon node slot_clock is within 1 slot, this is deemed acceptable. Otherwise the node
+/// will be marked as CandidateError::TimeDiscrepancy.
+const FUTURE_SLOT_TOLERANCE: Slot = Slot::new(1);
+
+// Configuration for the Beacon Node fallback.
+#[derive(Copy, Clone, Debug, Default, Serialize, Deserialize)]
+pub struct Config {
+    /// Disables publishing http api requests to all beacon nodes for select api calls.
+    pub disable_run_on_all: bool,
```

Comment:
The `disable-run-on-all` has been deprecated in favour of the `--broadcast` flag

Reply:
Fixed in [453e003](https://github.com/sigp/lighthouse/pull/4393/commits/453e003f926e27c09b5a04be7c18f7f6efd8f99f)